### PR TITLE
Add the breaker name to rejections

### PR DESCRIPTION
**Pull request by SimenB**
Not sure if you want this, but we find it practical when debugging and tracing through logs.

### CODE REVIEW

File: test/Circuit.spec.js [hunk 1]
```diff
@@ -191,6 +191,24 @@ describe('Circuit Class', () => {
     brake._circuitOpen = true;
     return circuit.exec('test').then(null, err => {
       expect(err).to.be.instanceof(CircuitBrokenError);
+      expect(err.message).to.equal('Circuit has been opened - The percentage of failed requests (NaN%) is greater than the threshold specified (50%)');
```
Comment (awolden) [inline test/Circuit.spec.js]: You can fix the `NaN%` message by making sure the brakes instance as registered at least one call `brake._failureHandler(100)`.

**Comment by awolden** [comment]
Overall the change is a good one 👍 . I am concerned that the change in the message output of the error may break peoples existing apps if they are looking at the current error message to determine error handling.

File: lib/Circuit.js [hunk 2]
```diff
@@ -84,6 +84,10 @@ class Circuit extends EventEmitter {
             return this._fallback.apply(this, arguments);
           }
 
+          if (err.message && this._brakes.name && this._brakes.name !== 'defaultBrake') {
```
Comment (awolden) [inline lib/Circuit.js]: We can remove `name !== 'defaultBrake'`. See other comment

**Comment by SimenB** [comment]
> the change in the message output of the error may break peoples existing

I agree it's a breaking change.

I'll fix the points you raised as soon as I get to work!

**Comment by SimenB** [comment]
@awolden Ping 😄

**Comment by awolden** [comment]
PR looks good :+1:, but I am hesitant to perform a major version bump for such a minor change. We will sit on this PR a bit and try to roll in with more changes.

**Comment by SimenB** [comment]
Any other planned breaking changes? If not, I suggest just merging and releasing this. 
A clear readme would mitigate the churn, I think. And a major is just a number :smile:

**Comment by awolden** [comment]
Hey SimenB,

Sorry for the delay. I don't want to push out a breaking change to aid in logging that can be accomplished pretty easily by the end consumer of the library. For instance you could do:

```javascript
const brake = new Brakes();

brake.request('foo')
  .catch(err => {
    console.error(`[Breaker: ${brake.name}] ${err.message}`);
  });

```

I would like it if Brakes handled this automatically for the user, but I also want to avoid version churn. While it is just a number, I don't want to increment a major version just for a change to add in logging.

**Comment by SimenB** [comment]
@awolden, yeah, that's what we do now 😄

**Comment by SimenB** [comment]
@awolden it's been over a year, could you merge and release this?

**Comment by awolden** [comment]
Yes, let's merge this. Can you fix the conflict on your end?

**Comment by SimenB** [comment]
Done!

**Comment by SimenB** [comment]
Woo! 🎉 Looking forward to release

**Pull request merged**